src/harness/tests/testdata/registration/reg2_a_singlestep.json

```diff
@@ -0,0 +1,21 @@
+{

```

Comment:
Yes. I prefer that idea to avoid creating similar json files. Otherwise people might end up with creating tons of json files.